app/Domain/Comments/Templates/submodules/generalComment.sub.php

```diff
@@ -157,28 +166,40 @@ class="deleteComment formModal">
 
     leantime.editorController.initSimpleEditor();
 
-    function toggleCommentBoxes(id, formHash) {
-
+    function toggleCommentBoxes(id, commentId, formHash,editComment = false, isReply = false) {
         <?php if ($login::userIsAtLeast($roles::$commenter)) { ?>
-            if (id == 0) {
-                jQuery('.mainToggler-'+formHash).hide();
+
+
+            if (parseInt(id, 10) === 0) {
```

Comment:
Radix is 10 by default, so the second parameter in parseInt() can be removed.

Reply:
Yeah true, I added it though, because if ever a linter is setup on this project it would probably complain about this.